Formula/openjdk.rb

```diff
@@ -0,0 +1,68 @@
+class Openjdk < Formula
+  desc "Development kit for the Java programming language"
+  homepage "https://openjdk.java.net/"
+  url "https://hg.openjdk.java.net/jdk-updates/jdk13u/archive/jdk-13.0.1+9.tar.bz2"
+  version "13.0.1+9"
+  sha256 "97328e767bc5f47b097ec0e9d88a6a650e60c448dbaba2e835284a2bf5594eb5"
+
+  keg_only :provided_by_macos
+
+  depends_on "autoconf" => :build
+  depends_on "openjdk@12" => :build
+  depends_on "freetype"
+
+  def install
+    ENV["JAVA_HOME"] = Formula["openjdk@12"].libexec/"openjdk-12.jdk/Contents/Home"
+    java_options = ENV.delete("_JAVA_OPTIONS")
+
+    inreplace "make/autoconf/lib-freetype.m4",
+      "AC_MSG_ERROR([Only bundled freetype can be specified on Mac and Windows])", ":"
+
+    short_version, _, build = version.to_s.rpartition("+")
+
+    chmod 0755, "configure"
+    system "./configure", "--without-version-pre",
+                          "--without-version-opt",
+                          "--with-version-build=#{build}",
+                          "--with-boot-jdk=#{ENV["JAVA_HOME"]}",
+                          "--with-boot-jdk-jvmargs=#{java_options}",
+                          "--with-debug-level=release",
+                          "--with-native-debug-symbols=none",
+                          "--enable-dtrace=auto",
+                          "--with-jvm-variants=server",
+                          "--with-freetype-include=#{Formula["freetype"].include}",
+                          "--with-freetype-lib=#{Formula["freetype"].lib}"
+
+    ENV["MAKEFLAGS"] = ENV["MAKEFLAGS"]&.sub("-j", "JOBS=")
+    system "make", "images"
+
+    libexec.install "build/macosx-x86_64-server-release/images/jdk-bundle/jdk-#{short_version}.jdk" => "openjdk-13.jdk"
+    bin.install_symlink Dir["#{libexec}/openjdk-13.jdk/Contents/Home/bin/*"]
+  end
+
+  def caveats
+    <<~EOS
+      To use this JDK, symlink it with
+
+        sudo ln -sfn #{libexec}/openjdk-13.jdk /Library/Java/JavaVirtualMachines/
+
+      or set JAVA_HOME:
+
+        export JAVA_HOME="#{libexec}/openjdk-13.jdk/Contents/Home"
```

Comment:
How does this compare to what the cask currently does?

Reply:
The casks install directly to `/Library/Java/JavaVirtualMachines`.

Reply:
Removed the `JAVA_HOME` part for now in any case to fix the audit error.